packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/integ.start-job-run.ts

```diff
@@ -100,6 +100,13 @@ new cdk.CfnOutput(stack, 'stateMachineArn', {
 
 new integ.IntegTest(app, 'aws-stepfunctions-tasks-emr-containers-start-job-run', {
   testCases: [stack],
+  cdkCommandOptions: {
+    deploy: {
+      args: {
+        rollback: true,
+      },
+    },
+  },
```

Comment:
3 tests were in need of update in stepfunctions-tasks, because they use a CDK construct that depends on lambda-layer-kubectl, and we updated the asset hash for the layer. `cdk deploy` has a normal default of `rollback: true`, but our test runner defaults to `rollback: false`. According to @corymhall, this is because of the assertion stack; we want them all to run even if some of them fail. There are no assertions here, so I feel safe turning on `rollback: true`.

However, calling this out: with `rollback: false`, the **update workflow** fails with `Replacement type updates not supported on stack with disable-rollback` on the `KubectlLayer`. It looks like on update, the different asset hash requires replacement of the lambda layer. And any sort of replacement is not allowed when `rollback: false`, a rule that comes from cloudformation and not the resource itself. I _think_ we should be okay with this, but I definitely want a second opinion. @madeline-k, @rix0rrr, what do you think?